js/ui/draggable.js

```diff
@@ -597,6 +604,7 @@ const Draggable = DOMComponent.inherit({
 
     _dragStartHandler: function(e) {
         const $element = this._getDraggableElement(e);
+        this.dragOccur = true;
```

Comment:
Let's rename it to the ```dragInProgress```

```suggestion
        this.dragInProgress = true;
```